contracts/test/RiscZeroGroth16Verifier.t.sol

```diff
@@ -138,6 +139,7 @@ contract RiscZeroGroth16VerifierTest is Test {
     }
 
     function testSelectorIsStable() external view {
-        require(verifier.SELECTOR() == hex"50bd1769");
+        IRiscZeroSelectable selectable = IRiscZeroSelectable(verifier);
+        require(selectable.SELECTOR() == hex"50bd1769");
```

Comment:
I don't think this change should be required, right? Because the `SELECTOR` function exists on `RiscZeroGroth16Verifier`